### DISCUSSION TITLE
Alexa non comunica più con HA

### DISCUSSION THREAD

**Discussion by vebto53** [indomus/forum#2525] [GENERALE - Discussioni generiche]
Ciao a tutti, uso da tempo HA core su raspberry con l'integrazione ad Alexa tramite skill haaska . Sempre funzionato bene con Alexa riuscivo a eseguire script e gestire entità varie (luci ecc). Da oggi Alexa non comanda più nessun dispositivo, dalla App  se tento l'attivazione , da tutti i device scrive "server non risponde" e non fa più niente.
Qualcuno può aiutarmi?
Grazie 
Pietro

**Comment by snakuzzo** [comment]
Fai il test della lambda function su AWS, e vedi cosa ti dice...

Vai [QUI](https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions), clicca sulla tua function e poi esegui il test...

![image](https://github.com/indomus/forum/assets/7205371/f116cf58-4929-48f5-b8be-26a0a8dbbb11)

Deve darti qualcosa del genere con `Status: Succeded`...
 
![image](https://github.com/indomus/forum/assets/7205371/38915e6c-fee8-4341-9656-b28ff05e09de)

Se qualcosa non va, il risultato sarà diverso e ti dirà cos'è che non va nel body della response

**Reply by vebto53** [reply]
Provato e mi da questo
![Schermata del 2023-07-03 23-29-16](https://github.com/indomus/forum/assets/83362338/2a118899-8aaa-43a7-a7fb-5e78a6563024)

**Reply by snakuzzo** [reply]
Vedo un timeout...sicuro che tu non abbia problemi a raggiungere HA da internet ?
controlla che la URL specificatanel file `config.json` in spalla sinistra sia corretta e che effettivamente da internet sia raggiungibile (con tanto di certificato valido)

**Reply by vebto53** [reply]
Dunque è successo questo: quando sono entrato in Aws per fare il test che mi avevi indicato, Aws non mi faceva accedere perché voleva che cambiassi la password. Ieri tarda serata ho fatto il change pw e il test che ho inviato sopra. Stamattina entro nella app alexa e vedo che tutti i dispositivi sono collegati e sembra tutto sia ritornato a funzionare. Mi vien da pensare che non collegasse la skill haaska perchè voleva il cambio password. Comunque sembra risolto. Grazie 1000 e a buon rendere. Pietro

**Reply by snakuzzo** [reply]
meglio così :) figurati
